File: source/Java/HouseholdData/src/model/SysDt.java
package model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the sys_dt database table.
 * 
 */
@Entity
@Table(name="sys_dt")
@NamedQuery(name="SysDt.findAll", query="SELECT s FROM SysDt s")
public class SysDt implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	private String kycd;

	private String dt;

	public SysDt() {
	}

	public String getKycd() {
		return this.kycd;
	}

	public void setKycd(String kycd) {
		this.kycd = kycd;
	}

	public String getDt() {
		return this.dt;
	}

	public void setDt(String dt) {
		this.dt = dt;
	}

}

File: source/Php/Household/Entity/SysDt.php
<?php
class SysDt {
	private $kycd;
	private $dt;
	public function setKycd($kycd) {
		$this->kycd = $kycd;
	}
	public function getKycd() {
		return $this->kycd;
	}
	public function setDt($dt) {
		$this->dt = $dt;
	}
	public function getDt() {
		return $this->dt;
	}
	public function toString(){
		return array(
			"kycd" => $this->kycd,
			"dt" => $this->dt
		);
	}
}
?>
